hedy.py

```diff
@@ -578,7 +591,34 @@ def elifs(self, args):
         all_lines = [indent(x) for x in args[1:]]
         return "\nelif " + args[0] + ":\n" + "\n".join(all_lines)
 
+class ConvertToPython_11(ConvertToPython_9):
+    def input(self, args):
+        var = args[0]
+        all_parameters = [a for a in args[1:]]
+        return f'{var} = input(' + '+'.join(all_parameters) + ")"
+
+class ConvertToPython_12(ConvertToPython_11):
```

Comment:
Hi! Since this function is useful for levels 12 and 13, I suggest renaming it to something that makes that explicit (for example: `ConvertToPython_12_13`).